src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -26,6 +26,10 @@
         overflow-y: auto;
         position: relative;
 
+        > .inline-alert-error {
+            margin: 9px 20px;
```

Comment:
This seems pretty arbitrary, any reason for 9px here?

Reply:
Maybe can extract these to variables since its being used in multiple places?

Reply:
This is the standard margin applied in the activity feed. Since the error isn't inside the activity feed, we need to apply the same margin so it matches the feed items style